spec-0000/index.md

```diff
@@ -42,6 +42,16 @@ For instance, if a newer minimum version of a core package is needed by a projec
 which is not backported to older versions.
 {{< /admonition >}}
 
+{{< admonition note >}}
+Certain projects (e.g., projects that have more resources) may wish to provide long-term support (LTS) of an additional year.
+
+Specifically, for projects wishing to provide LTS we recommend that:
```

Comment:
The first sentence is that all projects should adopt a common policy. This goes against that, so I think we should acknowledge it in some way. If we do not recommend one over the other, the first sentence might change to be "adopt one of two time-based poicies...". If we have a preference for the first but recognize the need for the other, we might acknowledge that here.